build/asset-manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "main.css": "static/css/main.2f893525.css",
```

Comment:
No need to add build files - they will be added automatically when pushed to master